### PR TITLE
[#165718205] Edit transaction feedback on wallet home

### PR DESCRIPTION
**Pull request by lmorelli986**
This PR aims to set the no-transaction feedback on wallet home as documentation

### iPhone X
iPhoneX

### iPhone 5s
iPhone5s

### Android
Android

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#165718205](https://www.pivotaltracker.com/story/show/165718205): Come CIT voglio avere chiaro dalla home del wallet se non ho transazioni o metodi di pagamento.

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by lmorelli986** [comment]
* Android's coming.
* Image missing. The one used in the screencast is a custom low quality inaccurate screenshot.

File: ts/screens/wallet/WalletHomeScreen.tsx [hunk 1]
```diff
@@ -59,10 +59,20 @@ const styles = StyleSheet.create({
 
   container: {
     flex: 1,
+    // alignContent: "center",
```
Comment (fpersico) [inline ts/screens/wallet/WalletHomeScreen.tsx]: No commented code.

Comment (lmorelli986) [reply ts/screens/wallet/WalletHomeScreen.tsx]: Sorry, I missed it out.

**Pull request merged**